README.md

```diff
@@ -1 +1,196 @@
-# Hello-world
\ No newline at end of file
+<a name="readme-top"></a>
+
+
+<div align="center">
+
+  <img src="murple_logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
```

Comment:
- [ ] you did a great job 😃 at using the microverse readme template, but kindly here try to remove the logo or add your logo here to make the Readme professional 😊